File: Pascal/1010/1010.pas
var
    code1, number1, code2, number2 : integer;
    price1, price2, total : double;
begin
    readln(code1, number1, price1);
    readln(code2, number2, price2);
    total := (number1*price1) + (number2*price2);
    writeln('VALOR A PAGAR: R$ ', total:2:2);
end.

File: C++/1010/1010.cpp
#include <cstdio>
using namespace std;

int code1, code2, number1, number2;
float price1, price2;

int main() {
    scanf("%d %d %f", &code1, &number1, &price1);
    scanf("%d %d %f", &code2, &number2, &price2);

    float total = (number1*price1) + (number2*price2);
    printf("VALOR A PAGAR: R$ %.2f\n", total);
    return 0;
}